runelite-client/src/main/java/net/runelite/client/plugins/config/PluginListPanel.java

```diff
@@ -101,6 +104,7 @@
 	private final JScrollPane scrollPane;
 	private final FixedWidthPanel mainPanel;
 	private List<PluginListItem> pluginList;
+	private final Table<String, String, Boolean> sectionExpandStates = HashBasedTable.create();
```

Comment:
Why can't this just be a `Map<ConfigSectionDescriptor, Boolean>`?
Also this should probably go in plugin, it has no relation to the state of the plugin list

Reply:
You did suggest `PluginListPanel` earlier, but yeah I get that.
Should I make it all static in ConfigPlugin or inject ConfigPlugin into ConfigPanel?

Reply:
Inject it

Reply:
Injecting it causes
```
java.lang.IllegalStateException: Recursive load of: net.runelite.client.plugins.config.PluginListPanel.<init>()```